pd-configuration-file.md

```diff
@@ -366,6 +366,11 @@ Configuration items related to replicas
 + See [Placement Rules](/configure-placement-rules.md).
 + An experimental feature of TiDB 4.0.
 
+### `flow-round-by-digit` <span class="version-mark">New in TiDB 5.1</span>
+
++ Default value: 3
++ PD will round the number of flow information to reduce the update of statistics caused by changes in Region flow information. This configuration item is used to specify the number of digits to round the Region flow information. For example, the flow `100512` will be rounded to `100500` because the default value is `3`. This configuration replaces `trace-region-flow`.
```

Comment:
```suggestion
+ PD will round the number of flow information to reduce the update of statistics caused by changes in Region flow information. This configuration item is used to specify the number of digits to round the Region flow information. For example, the flow `100512` will be rounded to `101000` because the default value is `3`. This configuration replaces `trace-region-flow`.
```
my fault, need fix.